model_analyzer/entrypoint.py

```diff
@@ -132,6 +133,8 @@ def get_server_handle(config, gpus):
             mounts=config.triton_docker_mounts,
             labels=config.triton_docker_labels)
     elif config.triton_launch_mode == 'c_api':
+        validate_triton_install_path(config)
```

Comment:
`validate_triton_install_path()` currently calls the `file_path_validator()`, which really just checks that a path exists

i think what `validate_triton_install_path()` should actually do is validate what the [triton_install_path option description](https://github.com/triton-inference-server/model_analyzer/blob/main/model_analyzer/config/input/config_command_profile.py#L581-L582) says:

> Path to Triton install directory i.e. the parent directory of 'lib/libtritonserver.so'. Required only when using triton_launch_mode=c_api.

i.e. `validate_triton_install_path()` should:

1. confirm that the value in the model analyzer config option `triton_server_path` is a path that exists and is a directory
2. confirm that `lib/libtritonserver.so` exists at the path and is a file, perhaps also check that it has executable permissions enabled. not sure if second part is necessary

also, perhaps `validate_triton_install_path()` can be moved to `model_analyzer/config/input/config_utils.py`

Reply:
1) Done.  
2) That specific check would couple MA to triton (not good - anytime they change the name of the file, the test would fail and we would have to change the name of the file along with them).  Instead, I added a test to make sure that at least one file exists.  Fair?

re `also, perhaps`: This function is pretty specific to this cli parameter.  To avoid clutter in config_utils.py, we can move this function to that class once another class needs this functionality.

Reply:
sounds good 👍